TiUP: Add three docs about tiup dm display, upgrade, and cluster check (#5291)
pingcap/docs: tiup/tiup-component-dm-upgrade.md

> --
> title: tiup dm upgrade
> --

Reviewer: ```suggestion
---
title: tiup dm upgrade
---
```

It needs three `-` to indicate this part as the metadata of the file.

# tiup dm upgrade

The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.

## Syntax 

```shell
tiup dm upgrade <cluster-name> <version> [flags]
```

> - `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, check it in [tiup DM cluster list](/tiup/tiup-component-dm-list.md).

Author: ```suggestion
- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, you can use the `[tiup dm list](/tiup/tiup-component-dm-list.md)` command to check it.
```

--
title: tiup dm upgrade
--

# tiup dm upgrade

The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.

## Syntax 

```shell
tiup dm upgrade <cluster-name> <version> [flags]
```

> - `<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).

Reviewer: ```suggestion
- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, check it in [tiup DM cluster list](/tiup/tiup-component-dm-list.md).
```

Author: @lucklove please help check if a similar change is also needed in the Chinese version. Thanks!

Reviewer 2: ditto

> - `<version>` is the target version to upgrade to. Currently, only upgrading to a later version is allowed, and upgrading to an earlier version is not allowed, which means the downgrade is not allowed. Upgrading to a nightly version is not allowed either.

Reviewer: ```suggestion
- `<version>` is the target version to be upgraded to. Currently, only upgrading to a later version is allowed, and upgrading to an earlier version is not allowed, which means the downgrade is not allowed. Upgrading to a nightly version is not allowed either.
```

## Options

### -h, --help

- Prints the help information.
- Data type: `BOOLEAN`
- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.

## Output

> Log of the upgraded service.

Reviewer: ```suggestion
Log of the service upgrade process.
```
